scripts/README.rst

```diff
@@ -91,10 +91,10 @@ Assuming you have the two prerequisites mentioned above, install the
 dependencies for the report generator, and run it, passing the following
 values on the command line:
 
-    * annotations_dir: path to where toggle data is location
-    * toggle_data_dir:  a path to directory containing directories containing json files with sql data dump
-    * output_path: path to where the reports will be written
-    * --show-state: if this is present, the report will include toggle state
+    * annotations_dir(required argument): path to where toggle data is location
```

Comment:
You might wNt to consider giving a working example and a —help example for details, and not repeating all the documentation here.